Gemfile

```diff
@@ -1,3 +1,4 @@
 source 'https://rubygems.org'
 
 gemspec
+gem "logstash-output-elasticsearch", :path => "../logstash-output-elasticsearch"
```

Comment:
Ah, oops! Errr, that should be removed. I only put it in because I depended
on my special branch (open in the other PR) for this to run

On Thu, Nov 12, 2015 at 3:49 AM, Suyog Rao <notifications@github.com> wrote:

> In Gemfile
> <https://github.com/logstash-plugins/logstash-output-elasticsearch_java/pull/29#discussion_r44637928>
> :
>
> > @@ -1,3 +1,4 @@
> >  source 'https://rubygems.org'
> >
> >  gemspec
> > +gem "logstash-output-elasticsearch", :path => "../logstash-output-elasticsearch"
>
> can we use LS_HOME here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/logstash-plugins/logstash-output-elasticsearch_java/pull/29/files#r44637928>
> .
>